helpers.py

```diff
@@ -103,5 +103,17 @@ def api_parameter_from_link(link):
         return None
 
 
+id_parser_regex = r'https?://[^/]+/[qa]/(\d+)'
```

Comment:
I believe we use `/questions/` URLs in a few places, I'm not sure what format we get here.

Reply:
@angussidney I must have forgot that. Changed it now.